services/lgtm/lgtm.tester.js

```diff
@@ -21,7 +21,7 @@ t.create('alerts: missing project')
   .get('/alerts/g/some-org/this-project-doesnt-exist.json')
   .expectJSON({
     name: 'lgtm',
-    value: 'project not found',
+    value: 'not found',
```

Comment:
One request from me: could you pass `errorMessages` when you make the request, so that the more detailed message is preserved?